apps/OboeTester/app/src/main/java/com/google/sample/oboe/manualtest/DeviceReportActivity.java

```diff
@@ -128,6 +130,27 @@ public String reportAllMicrophones() {
         return report.toString();
     }
 
+    private String getExtraDeviceInfo() {
+        StringBuffer report = new StringBuffer();
+        report.append("\n############################");
+        report.append("\nExtras:\n");
+        String unprocessedSupport = mAudioManager.getParameters(AudioManager.PROPERTY_SUPPORT_AUDIO_SOURCE_UNPROCESSED);
+        report.append("\nSUPPORT_UNPROCESSED : " + ((unprocessedSupport != null) ? "yes" : "no"));
```

Comment:
Since "no" could be incorrectly construed as Unprocessed is unavailable, could we have something like "SUPPORT_UNPROCESSED : full/partial"? We still should have "no" in case of pre-Android 7 as Unprocessed is actually not available there.

Reply:
Thanks. I want to display what the device is reporting without too much interpretation.
So I changed it to yes/null.